.github/workflows/full-loop.yml

```diff
@@ -16,7 +16,17 @@ jobs:
         with:
           fetch-depth: 0
       - uses: ./
+        name: Run action on full security-action repo
+        id: action
         with:
           debug: true
           github_token: ${{ secrets.GITHUB_TOKEN }}
           baseline_scan_only: false
+      - run: |
+          echo ${{ steps.action.outputs.reviewdog-findings }}
+          if [[ "${{ steps.action.outputs.reviewdog-findings }}" -lt "70" ]]; then
```

Comment:
this hardcoded 70 seems arbitrary

Reply:
The correct answer today (without brakeman) is 85. I don't know what to do. If safesvg stops working then it won't actually fail, as that's only a few findings, but if semgrep fails then it will show up.